src/App.js

```diff
@@ -29,7 +29,7 @@ class App extends Component {
   render() {
     const { user, token } = this.props;
     if (!user && token !== null) {
-      LocalApi.get("/users/me")
+      LocalApi.get("/users/me") //Bug here: need to fix in the future
```

Comment:
There's bug about this on the console. I'm not sure what you wanna achieve here, so I don't know if I can fix this. Let's talk about this tmr & see if we can figure it out :)

Reply:
I'll have a look, thanks